src/main/webapp/WEB-INF/hw/task1/chemicalElements.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html xmlns:margin="http://www.w3.org/1999/xhtml">
+    <head>
+        <style>
+            div {
+                width: 30px;
+                margin-left: auto;
+                margin-right: auto;
+            }
+            a {
+                color: blue;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>
+            Chemical elements
+        </h1>
+        <hr>
+        <table>
+            <tr>
```

Comment:
Let's add `thead`

Reply:
Ok, it's not a problem. I did it like `<tr>` because according the task column headers don't look bold: 
<img width="419" alt="image" src="https://github.com/Alexander-Khamitski/TMS_Java_EE/assets/93436427/1dd9d63e-22e4-4b66-b08b-76cc32fda136">

Reply:
@DHrytsuk Updated: [7cc84ffc](7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)